packages/react/src/ReactElementValidator.js

```diff
@@ -327,7 +327,10 @@ export function createElementWithValidation(type, props, children) {
     info += getStackAddendum() || '';
 
     let typeString;
-    if (type === null) {
+    if (type && type.$$typeof) {
```

Comment:
Let's move this into the last `else` clause instead?
Also let's make the check more concrete, e.g. `type.hasOwnPropery('$$typeof')`.

Reply:
Definitely will use `hasOwnProperty` over dot notation.

Just to be clear, are you saying to nest the `if` statement in that last `else` clause or make it the last `else if` before the `else` below?

Reply:
Going to go with the latter to avoid any nested `if` statements (ugly). If you want it the other way just lmk.